packages/zowe-explorer/src/dataset/actions.ts

```diff
@@ -303,7 +263,7 @@ export async function deleteDatasetPrompt(
     const deleteButton = localize("deleteDatasetPrompt.confirmation.delete", "Delete");
     const message = localize(
         "deleteDatasetPrompt.confirmation.message",
-        "Are you sure you want to delete the following {0} item(s)?\nThis will permanently remove these data sets and/or members from your system.\n\n{1}",
+        `Are you sure you want to delete the following {0} item(s)?\nThis will permanently remove these data sets and/or members from your system.\n\n{1}`,
```

Comment:
Should this string be split across multiple lines to make the linter happy?

Reply:
Just discussed w/ @t1m0thyj that the localize functions do not like any parameters other than raw string literals; unfortunately backticks introduce unwanted spaces in the localized strings, so that approach will not work either. Disabled lint warnings for the lines mentioned.